iOS-TvShows/iOS-TvShows/Modules/Shows/ShowsInterfaces.swift

```diff
@@ -14,16 +14,18 @@ import RxCocoa
 
 protocol ShowsWireframeInterface: WireframeInterface {
     func goToSettings()
+    func goToShowDetails(with model: Show)
 }
 
-protocol ShowsViewInterface: ViewInterface {
+protocol ShowsViewInterface: ViewInterface, Progressable {
 }
 
 protocol ShowsPresenterInterface: PresenterInterface {
     func configure(with output: Shows.ViewOutput) -> Shows.ViewInput
 }
 
 protocol ShowsInteractorInterface: InteractorInterface {
+    var shows: Single<[Show]> { get }
```

Comment:
Even though this is appealing to use due to the shorter syntax and not using () on a call site you are misleading the clients of this property - behind doors this property does some setup on each invocation - creates router, invokes service... Method would be a better fit here